### DISCUSSION TITLE
how to disable debug cmd window for release builds

### DISCUSSION THREAD

**Discussion by schleyk** [rustdesk/rustdesk#633] [Q&A]
Hello together,

I build the binary with "cargo build --release", can someone tell me how to turn off the second debug cmd window for the release builds?:

![image](https://user-images.githubusercontent.com/17356113/170711003-7c9ed99f-d361-4122-881c-fd244753f328.png)

Thanks! 👍

**Comment by sj6219** [comment]
The src/main.rs file contains the following parts:

#![windows_subsystem = "windows"]

You can uncomment.
